data/transactions/logic/eval_test.go

```diff
@@ -5527,3 +5527,33 @@ func TestTypeComplaints(t *testing.T) {
 	testProg(t, "err; store 0", AssemblerMaxVersion)
 	testProg(t, "int 1; return; store 0", AssemblerMaxVersion)
 }
+
+func TestSwitchInt(t *testing.T) {
```

Comment:
Let's make sure we have a test that jumps backward.

Let's confirm that the second one panics even if `err` is not on line 5554 (unless you've change the semantics to flow through when there's no match, which I maybe actually prefer. But if you did that, let's have a test that succeeds on flow through.

Reply:
I didn't change the semantics and have no idea why I put the `err` line in the test code. Still works when it's removed. Doesn't the first test here jump backwards? The intended behavior is for it to jump to `start` the first time `switch` is reached then jump to `end` the second time.

Reply:
Yes, it seems to do that, I totally missed that. But I also note that if the code is broken, and the first switch jumps to `end` instead of `start`, this would also accept.  I think I'd like to see some _brutally_ simple cases that just jump to a label, and that label has `int 1; return`, while other labels have `int 0; return`.  And have that for forward and backward jumps.

Ideally I'd like to know that the reference adjusting is working right. I think it is, because after optimization, the `int` refs becomes `intc0`, I think, thereby adjusting down in size.

How about a test with 0 labels?  Should that assemble?  I suppose not.  If you create it anyway in bytecode, what happens?

More than 256 labels?